packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -9,14 +9,15 @@ relatedComponents: ['task-list']
 
 ```jsx live
 <ProgressIndicator
+	activePath="/organisations"
 	items={[
-		{ label: 'Introduction', status: 'done', href: '#' },
-		{ label: 'Submit evidence', status: 'saved', href: '#' },
-		{ label: 'Organisations', status: 'started', href: '#' },
-		{ label: 'Business contacts', status: 'error', href: '#' },
-		{ label: 'Case studies', status: 'todo', href: '#' },
-		{ label: 'Attachments', status: 'started', href: '#', isActive: true },
-		{ label: 'Review and submit', status: 'blocked', href: '#' },
+		{ label: 'Introduction', status: 'done', href: '/introduction' },
```

Comment:
To prevent actually navigating away from the page, examples like SubNav put `#` at the start of the href url so it becomes a harmless hash link that still changes the path in the URL bar.

So for this it would look like 
`href: '#/introduction'`

Reply:
Hmm I think it's better to demonstrate how to use the component with folks' most common usage patterns. You can always click the back button to recover. Plus, if you click a different item and the state doesn't update it just seems broken. Plus plus, I'm wary of saying "it's totally fine to use in page links with this component".

What do you think about adding a separate example showing how to rig up `activePath` management, using the `#/introduction` technique you've mentioned and explicitly calling out not to do this in your app?

Reply:
SideNav examples currently use hash links

Reply:
Looks like I'm in the minority here. Just confirming, we'll go to hash links as per @Dan503 's suggestion?

Reply:
I would just do `#introduction` without the slash to match our other component examples. The slash feels weird

Reply:
- [ ] Move to hash links e.g. `#introduction`